pkg/fatfs/Makefile

```
@@ -2,7 +2,7 @@ PKG_NAME=fatfs
 # upstream server is very unreliable, instead host the extracted
 # .zip file on GitHub and apply only the upstream patches
 PKG_URL=https://github.com/RIOT-OS/FatFS
-PKG_VERSION=fb99d00924fd17c2d9c0789510852c2c286403ab # R0.14a
+PKG_VERSION=r0.14b
```

Better use the hash and put the version label as comment (the CI prefers that). 

```suggestion
PKG_VERSION=246c97b1eb7f1900eb00d5ed7e3bf4cdb4558039  # r0.14b
```